tests/approaches/test_base_approach.py

```diff
@@ -28,9 +28,7 @@ def _solve(self, task: Task, timeout: int) -> Callable[[State], Action]:
 
 def test_base_approach():
     """Tests for BaseApproach class."""
-    utils.update_config({
-        "seed": 123,
-    })
+    utils.reset_config({})
```

Comment:
Why do we need calls like this? Does it just set you back to the default CFG that's in settings.py? If so, maybe we should make the argument be optional (default None) and turn calls like this into `utils.reset_config()`.

Reply:
should be quick to do with a grep